### PR TITLE
creating/updating new user for SSH integration

### PR DESCRIPTION
**Pull request by joshpedroAL**

### CODE REVIEW

File: modules/users/manifests/joshuapedro.pp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Creates the joshuapedro user
 class users::joshuapedro {
   govuk_user { 'joshuapedro':
+    ensure   => absent
```
Comment (rtrinque) [inline modules/users/manifests/joshuapedro.pp:4]: missed a comma

**Review by rtrinque — CHANGES_REQUESTED** [review]

File: modules/users/manifests/joshuapedro.pp (hunk 1 above)
Comment (rtrinque) [inline modules/users/manifests/joshuapedro.pp:4]:
```suggestion
    ensure   => absent,
```

**Review by rtrinque — CHANGES_REQUESTED** [review]

**Review by joshpedroAL — COMMENTED** [review]
fixed comma issue

File: modules/users/manifests/joshpedro.pp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Creates the joshpedro user
+class users::joshpedro {
```
Comment (rtrinque) [inline modules/users/manifests/joshpedro.pp:2]: Uh, looks like computer said no to your choice of username :(

users::joshpedro should have a username of the correct form
     Failure/Error: expect(user[:name]).to eq(uname_from_full), "expected username '#{user[:name]}' to be '#{uname_from_full}' based on fullname and email"
       expected username 'joshpedro' to be 'joshuapedro' based on fullname and email

**Pull request closed without merging**